lib/generate-sitemap.js

```diff
@@ -0,0 +1,87 @@
+const url = require('url');
+const sitemapGenerator = require('sitemap');
+
+const register = require('../fixtures/register.json');
+const Fixture = require('../lib/model/Fixture.js');
+
+module.exports = function generateSitemap(options) {
+  const urls = getStaticUrls()
+    .concat(getRegisterUrls());
+  
+  const sitemap = sitemapGenerator.createSitemap({
+    hostname: url.resolve(options.url, '/'),
+    urls: urls
+  });
+
+  return sitemap.toString();
+};
+
+function getStaticUrls() {
+  return [
+    {
+      url: '/',
+      changefreq: 'daily'
+    },
+    {
+      url: 'fixture-editor/',
```

Comment:
We never link to pages with a trailing slash! But maybe you need it as a leading slash instead? If I have a *host* variable, either `host + '/'` or `host + 'fixture-editor/'` is wrong:

`...herokuapp.com/` vs. `...herokuapp.com//`  
`...herokuapp.comfixture-editor/` vs. `...herokuapp.com/fixture-editor/`

Reply:
One should provide trailing slashes if possible: https://www.sitemaps.org/protocol.html#locdef

I guess the sitemap dependency uses `path.join(...)`, that's why the double slashes don't matter:
```js
> path.join('a', 'b')
'a/b'
> path.join('a/', '/b')
'a/b'
```

I could use '' for the home page or use leading slashes everywhere to be consistent, though.

Reply:
> [...] and end with a trailing slash, **if your web server requires it**.

Ours doesn't 😉

The home page is actually referenced everywhere as `/` while all other pages have a leading slash and no trailing one. Please follow our previous conventions here.

Reply:
Done.